From c1c7f03db8416f489d076afb0d6fc17b2272b5c5 Mon Sep 17 00:00:00 2001
From: Salvador Teran Ochoa <salvador.teran.ochoa@intel.com>
Date: Tue, 2 Apr 2019 11:41:43 +0000
Subject: [PATCH] change install in /usr/etc to /usr/share/etc

Signed-off-by: Salvador Teran Ochoa <salvador.teran.ochoa@intel.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 93673d8..cbc8d6a 100644
--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,7 @@ LIBTOOL	:= libtool
 INSTALL	:= install
 
 DESTDIR	:= /
-PREFIX	:= /usr
+PREFIX	:= /usr/share
 
 all: audisp-json
 
-- 
2.21.0

